doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -20,28 +20,8 @@ Backwards-incompatible changes to these tables may be made at any time.
 reference these objects is not allowed.
 {{< /warning >}}
 
-### `mz_arrangement_sharing`
-
-The `mz_arrangement_sharing` view describes how many times each [arrangement]
-in the system is used.
 
-Field         | Type       | Meaning
---------------|------------|--------
-`operator_id` | [`bigint`] | The ID of the operator that created the arrangement. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).
-`worker_id`   | [`bigint`] | The ID of the worker thread hosting the arrangement.
-`count`       | [`bigint`] | The number of operators that share the arrangement.
-
-### `mz_arrangement_sizes`
-
-The `mz_arrangement_sizes` view describes the size of each [arrangement] in
-the system.
-
-Field         | Type       | Meaning
---------------|------------|--------
-`operator_id` | [`bigint`] | The ID of the operator that created the arrangement. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).
-`worker_id`   | [`bigint`] | The ID of the worker thread hosting the arrangement.
-`records`     | [`bigint`] | The number of records in the arrangement.
-`batches`     | [`bigint`] | The number of batches in the arrangement.
+## System Relations
```

Comment:
Not sure if this is a good heading. I added "Replica Introspection Sources" below and it felt weird to not also have a heading on the same level for the rest of the page. I just wasn't sure how to call these "relations that are not introspection relations", so I ended up with this overly generic term.